datacite-doi-commons/src/test/java/no/unit/nva/doi/DynamoRecordDtoMapperTest.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.doi;
+
+import static no.unit.nva.hamcrest.DoesNotHaveNullOrEmptyFields.doesNotHaveNullOrEmptyFields;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.PublicationDtoTestDataGenerator;
+import no.unit.nva.transformer.dto.DynamoRecordDto;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class DynamoRecordDtoMapperTest {
```

Comment:
ditto above about abstraction leak, discuss in comment further up.

Reply:
Now using DataCiteMetadataDto.